lib/commonConfig.js

```diff
@@ -245,6 +245,21 @@ function getCommandRegistry() {
     return commandRegistry;
 }
 
+/**
+ * It checks if the configuration file states the use of NGSIv3
```

Comment:
NGSIv3 -> NGSIv2

Reply:
Fixed in e9f6475e7ed413e8ac2e192187ffbd45399b5103